src/components/ChatEntry.js

```diff
@@ -1,22 +1,74 @@
 import React from 'react';
-import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
+import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({
+  id,
+  sender,
+  body,
+  timeStamp,
+  liked,
+  onUpdateMessages,
+  incrementCount,
+  decreaseCount,
+}) => {
+  const entryTime = TimeStamp(timeStamp);
+
+  let align = '';
+  if (id % 2 === 0) {
+    align = 'chat-entry remote';
+  } else {
+    align = 'chat-entry local';
+  }
```

Comment:
Great job taking on one of the optional enhancements!

Reply:
Thank you so much for reviewing my code Nancy!